client/pages/settings/workspace.vue

```diff
@@ -58,6 +58,69 @@
         </div>
       </div>
 
+      <div class="mt-2">
+        <h4 class="font-bold">Members</h4>
+        <form v-if="isWorkspaceAdmin" @submit.prevent="addUser" class="">
+          <div class="flex flex-col gap-2 mt-2">
+            <label for="newUser" class="text-sm font-medium text-gray-700">Email</label>
+            <input
+              v-model="newUser"
+              type="text"
+              id="newUser"
+              class="w-full p-2 border border-gray-200 outline-none rounded"
+              placeholder="Add a new user by email"
+              required
+            />
+          </div>
+          <div class="flex flex-col gap-2 mt-2">
+            <label for="newUserRole" class="text-sm font-medium text-gray-700">Role</label>
+            <select
+              v-model="newUserRole"
+              id="newUserRole"
+              class="w-full p-2 border border-gray-200 outline-none rounded"
+            >
+              <option value="user">User</option>
+              <option value="admin">Admin</option>
+            </select>
+          </div>
+          <div class="flex justify-end mt-2">
+            <v-button
+            color="outline-blue"
+            :loading="loadingUsers"
+          >
+            Add User
+          </v-button>
+          </div>
+        </form>
+        <UTable
```

Comment:
This table (and the way the data is loaded should go into a dedicated component). I think it's ok not use a dedicated store for this table since we'll only use the workspace users data here, but at least we need to group into logical component instead of putting everything in the page